app/views/mailer/user_device_new/de.html.erb

```diff
@@ -1,19 +1,19 @@
 #{config.product_name}-Anmeldung erfasst von einem neuen Gerät
 
-<div>Hallo #{user.firstname},</div>
+<div>Hallo #{user.firstname} #{user.lastname},</div>
```

Comment:
JFI: new device notifications are only available for agents. It's not available for customers.